src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/events/EventListener.kt

```diff
@@ -51,15 +52,16 @@ class EventListener(
   private fun getOffenderEvent(requestJson: String): OffenderEvent? {
     var event: OffenderEvent? = null
     try {
-      val message: Map<String, String?> = objectMapper.readValue(requestJson, Map::class.java) as Map<String, String?>
-      if (message == null || message["Message"] == null) {
+      val message: Map<String, Any?> = objectMapper.readValue(requestJson)
+      if (message["Message"] == null) {
         log.warn(requestJson)
       } else {
-        log.debug(message["Message"]) // do not log an excessive amount of data
-        event = objectMapper.readValue(message["Message"], OffenderEvent::class.java)
+        val body = message["Message"] as String
```

Comment:
Is there a chance of logging pii here or is it just the high level message event?

Reply:
sensitive info? I dont think so, its just incidents and alert info with codes , ids and nomsIds